build.sh

```diff
@@ -135,7 +135,7 @@ if [ $# == 1 ] ; then
     BUILD_FE=1
     BUILD_BE=1
     BUILD_BROKER=1
-    BUILD_META_TOOL=1
+    BUILD_META_TOOL=ON
```

Comment:
sometimes we use on, sometimes we use 1, what's the difference?

Reply:
ON is pass to Cmake, 1/0 is used by the bash script